src/controller/AppHandle.cpp

```diff
@@ -27,6 +27,23 @@ AppHandle::AppState AppHandle::start()
     setStateAndNotify (process.start (config.startCommand, streamFlags) ? AppState::alive : AppState::startFailed);
     startTimerHz (stateUpdateHz);
 
+    // Start reading the output asynchronously as it is blocking.
+    if (streamFlags)
+    {
+        auto readOutput = [&] {
+
+            char buffer[512];
+
+            while (auto numRead = process.readProcessOutput (buffer, sizeof (buffer)))
+            {
+                juce::ScopedLock sl (outputLock);
+                output.write (buffer, size_t (numRead));
+            }
+        };
+
+        juce::Thread::launch (std::move (readOutput));
```

Comment:
I would consider using a `std::thread` instead of `juce::Thread::launch` as it returns no handle to the thread